README.md

```diff
@@ -1,78 +1,104 @@
+### Subscription scenario
+
+1. Customer Service Representative (CSR) creates a new product and defines auto-order schedule for it on admin side;
+![Fig. Autoreorder schedule](docs/media/screen-autoorder-schedule.png)
+1. Customer orders the product on storefront
+1. The system displays the subscription schedule under the product;
+![Fig. Subscription period](docs/media/screen-subscription-period.png)
+1. There will be no possibility to select "I want this to be a recurring order" during checkout.
+1. The following steps are identical to the ones in "Recurring order scenario" starting from step 5.
 
 ![Subscription workflow](https://cloud.githubusercontent.com/assets/5801549/21925030/47b98646-d984-11e6-82db-0d89cc686efa.png)
 
+### Subscription states
 
-## Subscription states
 Subscription "UML state machine" diagram:
 
 ![Subscription state machine](https://cloud.githubusercontent.com/assets/5801549/21925541/9d415b82-d986-11e6-98db-2c4487279019.png)
 
+### Data flows
 
-## Data flows
 Subscriptions are used for generating new Orders under some rules. That is why we always store 2 entities: Subscription and associated Order. The latter we call a "prototype Order" as future Orders will be generated from it.
 
-* When a Customer creates an Order in storefront, it's sent to Orders service;
-* Orders service saves the Customer Order to database and also fires OrderChangeEvent even;
-* Subscription module receives OrderChangeEvent (containing the Order).
-    * If this was a newly created Order with associated payment plan (and not a prototype for some Subscription), then:
-    	1. Create new Subscription;
-    	2. Clone given Order and associate it with the new Subscription.
-    * If the received Order was generated by a Subscription, then the Subscription should be updated (status, balance recalculated, etc.). This includes the scenario when new Order was generated by Process Subscription Job.
-* Scheduled job initiates Subscription processing periodically. It reads active Subscriptions from database and generates new Orders (if needed). 
+1. When a Customer creates an Order on storefront, it's sent to Orders service;
+1. Orders service saves the Customer Order to database and also fires; OrderChangeEvent even;
+1. Subscription module receives OrderChangeEvent (containing the Order);
+1. If this was a newly created Order with associated payment plan (and not a prototype for some Subscription), then:
+
+     1. Create a new Subscription-
+    clone given Order and associate it with the new Subscription;
+     1. If the received Order was generated by a Subscription, then the Subscription should be updated (status, balance recalculated, etc.). This includes the scenario when new Order was generated by Process Subscription Job.
+
+1. Scheduled job initiates Subscription processing periodically. It reads active Subscriptions from database and generates new Orders (if needed).
```

Comment:
Describe more about scheduled job

Reply:
Please give me info, what exactly we should describe ?

Reply:
This is the first place in the article where “scheduled job” is mentioned. It is not clear who and when “scheduled job” created.